docs/src/main/asciidoc/hibernate-orm-panache.adoc

```diff
@@ -286,21 +290,28 @@ public class Person extends PanacheEntity {
 Normally, HQL queries are of this form: `from EntityName [where ...] [order by ...]`, with optional elements
 at the end.
 
-If your query does not start with `from`, we support the following additional forms:
+If your select query does not start with `from`, we support the following additional forms:
 
 - `order by ...` which will expand to `from EntityName order by ...`
 - `<singleColumnName>` (and single parameter) which will expand to `from EntityName where <singleColumnName> = ?`
 - `<query>` will expand to `from EntityName where <query>`
 
+If your update query does not start with `update`, we support the following additional forms:
+
+- `from EntityName ...` which will expand to `update from EntityName ...`
+- `set <singleColumnName>` (and single parameter) which will expand to `update from EntityName set <singleColumnName> = ?`
+- `<singleColumnName>` (and single parameter) which will expand to `update from EntityName set <singleColumnName> = ?`
+- `<query>` will expand to `update from EntityName <query>`
```

Comment:
Hold on, what does it mean to have an update with a query and no `set` clauses?

Reply:
Oh, it's not a query, and it expands to `update from EntityName SET <query>`, so that doc (and perhaps the `package-info` one too`) is wrong: this is a shortcur to an assignment `foo = ...` clause.

Reply:
It should be something like:

- `from EntityName ...` which will expand to `update from EntityName ...`
- `set? <singleColumnName>` (and single parameter) which will expand to `update from EntityName set <singleColumnName> = ?`
- `set? <update-query>` will expand to `update from EntityName set <update-query>`

Reply:
Probably done. :)